discovery-provider/plugins/pedalboard/apps/sla-auditor/src/audits/version.ts

```diff
@@ -52,8 +55,8 @@ const getVersionData = async (
     console.log(`Caught error ${e} making request to ${node.endpoint}`);
     return {
       nodeEndpoint: node.endpoint,
-      nodeVersion: "",
-      minVersion: "",
+      nodeVersion: "N/A",
+      minVersion: minVersions[nodeType],
```

Comment:
the PR description says you can show the minVersion even without knowing the node type. not sure i follow how this works if nodeType is unknown, but also i'd think nodeType would never need to be unknown if reading from some RPC or gateway

Reply:
yeah comment is a little unclear. before this PR, node type was inferred from the request to the health check endpoint, rather than the "chain"